dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -201,16 +201,22 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/fuse/bin/alluxio-fuse",
 		"integration/kubernetes/alluxio-fuse.yaml.template",
 		"integration/kubernetes/alluxio-fuse-client.yaml.template",
-		"integration/kubernetes/alluxio-journal-volume.yaml.template",
```

Comment:
instead of deleting these, replace these with a symlink to the `singleMaster` configuration